cla-backend/cla/models/github_models.py

```
@@ -2405,51 +2369,46 @@ def get_pull_request_commit_authors(client, org, pull_request, installation_id,
-    if gql_ok:
+    if compare_ok:
         try:
             with concurrent.futures.ThreadPoolExecutor(max_workers=max_workers) as executor:
-                for chunk in iter_chunks(iter_pr_commits_full(client, org, repo_name, pr_number), submit_chunk):
+                for chunk in iter_chunks(iter_pr_commits_compare(client, org, repo_name, pr_number), submit_chunk):
                     futures = [executor.submit(get_author_summary_gql, c, pr_number, installation_id, with_co_authors) for c in chunk]
                     for fut in concurrent.futures.as_completed(futures):
                         authors, missing = fut.result()
                         any_missing = any_missing or missing
                         commit_authors.extend(authors)
         except Exception as exc:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)